wordpress-mobile/gutenberg-mobile
# VideoPress block: Add tests to verify Privacy and Rating settings

Part of the coverage efforts outlined in https://github.com/wordpress-mobile/gutenberg-mobile/issues/5734

## Description

* The changes in this PR iterate on the work done in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5751 to add test coverage for the Privacy and Rating settings. 
* A new helper, [`pressSettingInPicker`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/e46c8346c1670aed9190141987b6b4fd8296957c/src/test/videopress/local-helpers/utils.js#L62-L87), has been introduced to loop through and select each of the available options in the Privacy and Rating pickers.

## To test:

* Verify the HTML the changes to [the snapshot file](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5754/files#diff-e223c3972444b923239b4ee32c702e552bee64ddf0b437a84ed292a5416097b6) matches with the reality of what the block's HTML would look after making the relevant changes.
* Run the `TEST_RN_PLATFORM=ios npm run test src/test/videopress/edit.js` and `TEST_RN_PLATFORM=android npm run test src/test/videopress/edit.js` commands from the terminal to verify the tests pass as expected. Also, verify the tests on this PR pass.

<hr />

PR submission checklist:
